sunpy/util/util.py

```diff
@@ -179,12 +186,13 @@ def replacement_filename(path):
                 return newpath
 
 
-#==============================================================================
+# =============================================================================
 # expand list from :http://stackoverflow.com/a/2185971/2486799
-#==============================================================================
+# =============================================================================
 def expand_list(input):
```

Comment:
No idea - I didn't add that. I agree with you that we wouldn't need to use this.